Makefile

```diff
@@ -158,6 +163,14 @@ $(OUT)/test-%: tests/test-%.py $(OUT)/libdcurl.so
 	$(call py_prepare_cmd)
 	$(Q)chmod +x $@
 
+$(OUT)/%.o: remotedcurl/%.c
```

Comment:
Why did you create a new directory for `remotedcurl`? It can be consolidated with single source file.